public/style.css

```diff
@@ -776,25 +839,6 @@ section {
     animation-delay: 3s;
     animation-fill-mode: forwards;
   }
-
```

Comment:
Ik zie hier dat je dit hebt weggehaald of verplaatst? Dit betreft de stopmotion van het hartje in de succesmelding na het liken van een playlist. Ik zou deze graag willen houden dus graag even controleren of dit er nog staat of niet.